faults.md

```diff
@@ -10,43 +10,74 @@ A fault is what happens when partcipants in the protocol are behaving incorrectl
-  - Note: if a sector is missed here, and they are recovered after the fact, the miner could simple 're-commit' the sector. They still have to pay the collateral, but the data can be quickly re-introduced into the system to avoid clients calling them out for breach of contract (this would only work because the sector commD/commR is the same)
-  - Note: In the case where a miner is temporarily unable to prove some of their data, they can simply wait for the temporary unavailability to recover, and then continue proving, submitting the proofs a bit late if necessary (paying appropriate fees, as described above).
-- **Breach of contract dispute:**
-  - **Condition:** A client who has stored data with a miner, and the miner removes the sector containing that data before the end of the agreed upon time period.
-  - **Reporting:** The client invokes `ArbitrateDeal` on the offending miner actor with a signed deal from that miner for the storage in question. Note: the reporting must happen within one proving period of the miner removing the storage erroneously.
-  - **Check:** The chain checks that the deal was correctly signed by the miner in question, that the deal has not yet expired, and that the sector referenced by the deal is no longer in the miners proving set.
-  - **Penalization:** The miner is penalized an amount proportional to the incorrectly removed sector. This penalty is taken from their pledged collateral .
-  - Note: This implies that miners cannot re-seal data into different sectors. We could come up with a protocol where the client gives the miner explicit consent to re-seal, but that is more complicated and can be done later.
+  - **Penalization:**
+    - *Economic penalization*: Miner loses all pledge and storage collateral.
+    - *Power penalization*: The miners' power is reduced to `0`.
+  - **Recovery**: Only resubmission of the sectors can lead to recovery.
+  - *Note*: If a miner is in this state, where they have failed to submit a PoST, any block they attempt to mine will be invalid, even if the election function selects them. (the election function should probably be made to never select them)
+
+
+#### Reported storage fault penalty
+
+- **Condition:** The miner submits their PoSt with a non-empty set of 'missing sectors', on time.
+- **Reporting:** The miner can specify some sectors that they failed to prove during the proving period.
+  - *Note*: These faults are output by the `ProveStorage` routine, and are posted on-chain when posting the proof. This occurs when the miner (for example) has a disk failure, or other local data corruption.
+- **Check:** The chain checks that the proof verifies with the missing sectors.
+- **Penalization:** The miner is penalized pledge collateral, storage collateral and power proportional to the number of missing sectors. The sectors are also removed from the miners proving set.
+- **Recovery**:
+  - The faulted sectors must be re-submitted as if there were new sectors. 
```

Comment:
When re-sealing and re-submitting, can a miner reuse the id of a sector which it has permanently lost?